integration/spark2/src/main/scala/org/apache/spark/sql/hive/CarbonRelation.scala

```diff
@@ -172,7 +172,13 @@ case class CarbonRelation(
   }
 
   // TODO: Use data from the footers.
-  override lazy val statistics = Statistics(sizeInBytes = this.sizeInBytes)
+  // TODO For 2.1
+  //  override lazy val statistics = Statistics(sizeInBytes = this.sizeInBytes)
+  // Todo for 2.2
+  //  override def computeStats(conf: SQLConf): Statistics = Statistics(sizeInBytes =
+  //  this.sizeInBytes)
+
+  // override lazy val statistics = Statistics(sizeInBytes = this.sizeInBytes)
```

Comment:
It would be a big problem if we can't implement it. Please check it on priority

Reply:
Will resolve it as soon as possible.